### ISSUE TITLE
Config not applying properly; new plugins not installing

### ISSUE THREAD

**Issue by joeyfigaro** [zeit/hyper#1724]
- [X] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [X] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: 10.12.4 OS X
- **Hyper.app version**: 1.3.1
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: [.hyper.js](https://gist.github.com/joeyfigaro/d1929e3048f91ee3843cfbbea67ada10)
- **Relevant information from devtools** _(CMD+ALT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: `Warning: Unknown OSC code: "1" : bundle.js:47`
- **The issue is reproducible in vanilla Hyper.app**: Yup! Reinstalled a couple times. Hyper isn't honoring config.

## Issue
Hyper is not accepting my config updates properly. It reloads just fine, but my changes don't come through. I realize (though I don't understand why) that by using `hyper-snazzy`, I can't set my colors. Does my config not override that of a plugin's config? 

Also, adding a new plugin to `plugins []` results in a refresh in hyper with an error about not being able to find the module for the plugin that was just added. Looks like npm isn't actually installing anything when I add something new, so I have to do it by hand. :)

**Comment by chabou** [comment]
> Does my config not override that of a plugin's config?

Indeed, plugins override your config. It is a good design for plugins to let user override its configuration but some plugins don't offer this.

You can use: [hyperterm-final-say](https://www.npmjs.com/package/hyperterm-final-say) to define your config on top of plugins one. But use it carefully because it could break some plugins functionality (like [hyper-autoprofile](https://www.npmjs.com/package/hyper-autoprofile)).

I think we need more details about your plugins installation problem. Maybe restart Hyper could help? Do you use `nvm` or `n`? is this happening each time?

**Comment by joeyfigaro** [comment]
@chabou Thanks for jumping in, man. It *looks* like hyper is just reloading before it's done installing new dependencies. If I wait a few seconds and reload again, the error is gone and the new dependency can be found in `node_modules`... maybe the automatic refresh should happen after `node_modules` is updated.

**Comment by nesl247** [comment]
I too am having the issue of hyper not actually installing anything when changing the plugins array.
